cmd/okp4d/main.go

```diff
@@ -24,6 +24,13 @@ func main() {
 	rootCmd.Short = Resource.Short
 	rootCmd.Long = Resource.Long
 
+	for _, element := range rootCmd.Commands() {
```

Comment:
Just a small remark. You could put a comment to explain the process of replacing the existing CLI command with ours. (it's quite clear when reading the code, but it would be more explicit if you explain it).

Reply:
Yes, you're right, it will be more explicit, thanks. ✅